pkg/snapstore/snapstore_test.go

```diff
@@ -487,14 +546,42 @@ func modifyCredentialFileInDirectory(credentialDirectory, credentialFile string)
 	return os.WriteFile(credentialFilePath, []byte("MODIFIED CONTENT"), os.ModePerm)
 }
 
+// generates a unique string that would be the contents of a snapshot
+func generateContentsForSnapshot(snapshot *brtypes.Snapshot) string {
```

Comment:
Why do we need this, when we can write the actual snapshot contents to the snapshot object, like was done previously?

Reply:
The number of `expectedValX` variables would be a lot more since we need to create and hold the data that would be stored in the segment objects for Swift as well, and for each additional snapshot that is added to the tests in the future, an extra 2 `expectedValXSegmentObject` would need to be added.

Instead of just hardcoding a different value for each snapshot, since all the snapshots are unique, a string is just created on the fly and stored in the `objectMap` before the tests are started. When the `Fetch` is called, this string can be generated again to check the validity of the snapshots.

Doing this eliminates the burden of matching `snapX` to `expectedValX` both while reading the code and adding code.